### ISSUE TITLE
instructions for how student without $99/yr Apple developer account can run apps on their iPhones

### ISSUE THREAD

**Issue by kmolo** [mobilebridge/iosbridge#11]
I remember someone discussing the convoluted steps necessary to allow students with only free account to run apps on their phones (requiring Xcode 7+).  

@jeffreybergier was that you who mentioned this to me, or someone else?

Trying to find out who knows this, so this can be written up in MobileBridge documents...but also organizing a pilot CodePath University iOS bootcamp at CCSF, so these questions are coming up there.

**Comment by jeffreybergier** [comment]
Yeah, its kind of a wonky process. I did not run through it. I just typed these notes in my head. So it might be good to do it yourself when writing the steps so you can take screenshots and/or verify the names of things I mentioned are correct.

1. In Xcode go to Preferences > Accounts
2. Add developer account
3. In a sample iOS project, go to the main project editor (where you can set whether it runs on iPhone/iPad/OS Version, etc
4. In the Team drop down, choose your name (the developers name). Its likely the only item in the drop down
5. Now plug in the iOS device and select it in the scheme dropdown at the top left of the Xcode window.
6. Run. 
7. You'll probably get an error about certificates. Just click the Fix-it button. 
8. Now when you Run again you'll get an indecipherable error

This is where the confusing part happens. With these free certificates, there is an extra level of security that is not present with the paid ones. On the iOS device (after performing all the above steps)

1. Go to settings > General > Profiles
2. You should see a development or team certificate with the developers name. Tap on that
3. There should be an allow or approve button to approve apps from this developer

Now back in Xcode
1. Click run again

**Comment by kmolo** [comment]
Thanks!

Follow-up question: the apps that are loaded onto the student's iDevice after this process...do they still remain on their iDevice after they disconnect their iDevice from their computer?  Basically this is a process to side-load apps onto ones iDevice?  (Thinking of the security holes from loading sub-optimally coded apps on one's iDevice.)

**Comment by jeffreybergier** [comment]
They do :).

Once on there, they stay on there until they are manually deleted or the device is erased.
